PromiseTests/ExecutionContextTests.swift

```diff
@@ -10,6 +10,10 @@ import XCTest
 
 import Promise
 
+#if os(Linux)
+    import Dispatch
+#endif
+
```

Comment:
i think on mac you can call `import Dispatch` without any ill effects, so i think it's best to remove this conditional

Reply:
The premise is to align with the Ray Wenderlich Community Guidelines, as per https://github.com/raywenderlich/swift-style-guide#minimal-imports . The import is unnecessary on Apple-platforms and despite its ugly looks, pre-conditionals are a necessity recommended in most of Swift on Linux guides published by Apple. Should I still remove the pre-conditionals?

Reply:
part of the goal of this project is to keep things as simple as possible, and i think `os(Linux)` is added noise that isn't adding anything. 👍 to getting rid of them

Reply:
Okay